dash-pipeline/SAI/templates/saifixedapis.cpp.j2

```diff
@@ -55,8 +56,8 @@ sai_status_t sai_create_switch_dummy(
 #define DASH_BMV2_DEFAULT_1Q_BRIDGE_ID MAKE_OID(SAI_OBJECT_TYPE_BRIDGE,1)
 
 sai_object_id_t port_list[DASH_BMV2_NUM_PORTS] = {
-    MAKE_OID(SAI_OBJECT_TYPE_PORT,1),
-    MAKE_OID(SAI_OBJECT_TYPE_PORT,2)
+    (sai_object_id_t)0,
+    (sai_object_id_t)1
```

Comment:
since we have not fixed yet to make it work with $ PORT_0 why do we need to change this ?

Reply:
Mircea, this is a good idea to put in anyway. Once we solve the "object discovery" problem it should then provide the right values.

Reply:
i don't agree if this goes different from what we get on a switch , why make dash different than other SAI devices

Reply:
Presumably, a real switch will provide port OIDs which will then be discovered by the controller (SAI Challenger, PTF or NOS) and use them to populate subsequent SAI entry calls requiring a PORT OID. It's up to each target to provide a startup list of ports. Since DASH bmv2 libsai doesn't use real OIDs, this is an OK way to handle this.